modnet/featurizers/presets/debreuck_2020.py

```diff
@@ -113,44 +114,57 @@ class DeBreuck2020Featurizer(modnet.featurizers.MODFeaturizer):
     )
 
     def featurize_composition(self, df):
-        """ Applies the preset composition featurizers to the input dataframe,
+        """Applies the preset composition featurizers to the input dataframe,
         renames some fields and cleans the output dataframe.
 
         """
         df = super().featurize_composition(df)
 
         _orbitals = {"s": 1, "p": 2, "d": 3, "f": 4}
-        df['AtomicOrbitals|HOMO_character'] = df['AtomicOrbitals|HOMO_character'].map(_orbitals)
-        df['AtomicOrbitals|LUMO_character'] = df['AtomicOrbitals|LUMO_character'].map(_orbitals)
+        df["AtomicOrbitals|HOMO_character"] = df["AtomicOrbitals|HOMO_character"].map(
+            _orbitals
+        )
+        df["AtomicOrbitals|LUMO_character"] = df["AtomicOrbitals|LUMO_character"].map(
+            _orbitals
+        )
 
-        df['AtomicOrbitals|HOMO_element'] = df['AtomicOrbitals|HOMO_element'].apply(
+        df["AtomicOrbitals|HOMO_element"] = df["AtomicOrbitals|HOMO_element"].apply(
             lambda x: -1 if not isinstance(x, str) else Element(x).Z
         )
-        df['AtomicOrbitals|LUMO_element'] = df['AtomicOrbitals|LUMO_element'].apply(
+        df["AtomicOrbitals|LUMO_element"] = df["AtomicOrbitals|LUMO_element"].apply(
             lambda x: -1 if not isinstance(x, str) else Element(x).Z
         )
 
-        df = df.replace([np.inf, -np.inf, np.nan], 0)
```

Comment:
We should check this throughout the whole featurization process. Do we consistently replace the nan with some value? here it is zero, but in cleaning it is -1.

Reply:
(could potentially interfere with the test. at least small pre-featurized_df. But exclusively calling cleaning_df is indeed the most consitent way f doing things. Should preprocess the df in the same way for prediction.

Reply:
Agree, should raise an issue about this. I recommend keeping it the same for now (and for 0.1.9) but we may consider leaving the NaN's for a sklearn feature scaler later on.

Reply:
Opened #28 to track this, will resolve the comments here